src/System.Text.Json/src/System/Text/Json/Serialization/JsonCamelCaseNamingPolicy.cs

```diff
@@ -13,8 +13,21 @@ public override string ConvertName(string name)
                 return name;
             }
 
+#if BUILDING_INBOX_LIBRARY
+            return string.Create(name.Length, name, (chars, name) =>
+            {
+                name.AsSpan().CopyTo(chars);
+                FixCasing(chars);
+            });
+#else
             char[] chars = name.ToCharArray();
```

Comment:
Given property/key names are almost always small, is it worth it to copy the contents into a stackalloc span instead of always calling `ToCharArray` for say < 256?

Reply:
Maybe?  Feel free to look at that separately; I'm going to skip it for this PR.  I'm also not sure how much time we want to invest in the OOBs.